components/StatisticsCard.vue

```diff
@@ -1,15 +1,33 @@
 <template>
   <div class="statistics-card">
     <p class="statistics-card__text">{{ data.text }}</p>
-    <div class="statistics-card__bar"></div>
+    <div class="statistics-card__bar-container">
+      <progress-bar
+        v-if="data.barType === '1'"
+        :value="data.value"
+        :max-value="data.maxValue"
+      />
+      <progress-bar-double
+        v-if="data.barType === '2'"
+        :previous-value="data.previousValue"
+        :current-value="data.currentValue"
+        :max-value="data.maxValue"
+      />
+    </div>
     <p class="statistics-card__digits">{{ data.digits }}</p>
     <p class="statistics-card__source">{{ data.source }}</p>
   </div>
 </template>
 
 <script>
+import ProgressBar from '@/components/ui/ProgressBar';
+import ProgressBarDouble from '@/components/ui/ProgressBarDouble.vue';
 export default {
   props: ['data'],
+  components: {
+    'progress-bar': ProgressBar,
+    'progress-bar-double': ProgressBarDouble,
+  },
 };
 </script>
 
```

Comment:
```suggestion
.statistics-card {
  display: flex;
  flex-direction: column;
  align-items: flex-end;
  align-content: space-between;
  width: 100%;
  min-height: 300px;
  max-width: 300px;
  border: 1px solid #efefef;
  padding: 20px;
}
```

Делаем ячейки flex контейнерами. И убираем position.